bl_caster_l_wheel_joint: -634.2670155770062
bl_caster_r_wheel_joint: -543.0901144685564
bl_caster_rotation_joint: -15.618926303080464
br_caster_l_wheel_joint: -215.78920266019438
br_caster_r_wheel_joint: -45.58167743905368
br_caster_rotation_joint: 56.644001130622094
fl_caster_l_wheel_joint: -158.72071836594637
fl_caster_r_wheel_joint: -79.84016005377848
fl_caster_rotation_joint: -18.751400030876095
fr_caster_l_wheel_joint: -76.19814038771264
fr_caster_r_wheel_joint: -25.536692649487286
fr_caster_rotation_joint: -31.31364541690032
head_pan_joint: 0.06284408264213939
head_tilt_joint: 1.3158037230785251
l_elbow_flex_joint: -1.6573083107568838
l_forearm_roll_joint: -4.249298669798033
l_gripper_joint: 0.048743592246902154
l_gripper_l_finger_joint: 0.27478932824898683
l_gripper_l_finger_tip_joint: 0.27478932824898683
l_gripper_motor_screw_joint: 0.0
l_gripper_motor_slider_joint: 0.0
l_gripper_r_finger_joint: 0.27478932824898683
l_gripper_r_finger_tip_joint: 0.27478932824898683
l_shoulder_lift_joint: 0.41440381207787713
l_shoulder_pan_joint: 0.3739030489767273
l_upper_arm_roll_joint: 1.6492276731878892
l_wrist_flex_joint: -1.507108125978472
l_wrist_roll_joint: 3.3515352275642396
laser_tilt_mount_joint: -0.03123277389786054
r_elbow_flex_joint: -1.6778657517733446
r_forearm_roll_joint: -6.930921003036001
r_gripper_joint: 0.011424718045277806
r_gripper_l_finger_joint: 0.06482739394435197
r_gripper_l_finger_tip_joint: 0.06482739394435197
r_gripper_motor_screw_joint: 0.0
r_gripper_motor_slider_joint: 0.0
r_gripper_r_finger_joint: 0.06482739394435197
r_gripper_r_finger_tip_joint: 0.06482739394435197
r_shoulder_lift_joint: 0.5399020516322183
r_shoulder_pan_joint: -1.0072399522318864
r_upper_arm_roll_joint: -1.5927772458065141
r_wrist_flex_joint: -0.9326894549145393
r_wrist_roll_joint: -4.791633517559082
torso_lift_joint: 0.29976216761282093
torso_lift_motor_screw_joint: 0.0
